test-resources/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "one-login-test-resources",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "license": "MIT",
+  "scripts": {
+    "deploy": "cd ../helper-scripts && ./deploy_test_resources.sh",
+    "format": "./node_modules/.bin/prettier --write '**/*.ts'",
+    "format:check": "./node_modules/.bin/prettier --check '**/*.ts'",
+    "lint": "./node_modules/.bin/eslint ./dequeue",
+    "test:unit": "./node_modules/.bin/jest --testPathPattern=dequeue/ --silent"
+  },
+  "engines": {
+    "node": "20"
+  },
+  "devDependencies": {
+    "@eslint/js": "9.15.0",
+    "@types/aws-lambda": "8.10.146",
+    "@types/eslint__js": "8.42.3",
+    "@types/jest": "29.5.14",
+    "@types/js-yaml": "4.0.9",
+    "eslint": "9.15.0",
+    "jest": "29.7.0",
+    "jest-junit": "16.0.0",
+    "prettier": "3.3.3",
+    "ts-jest": "29.2.5",
+    "ts-node": "10.9.2",
+    "typescript": "5.7.2",
+    "typescript-eslint": "8.16.0",
+    "yaml": "2.6.1"
```

Comment:
Please may you double check this is used?

Reply:
Removed